cfripper/cli.py

```diff
@@ -138,6 +145,11 @@ def process_template(
     help="Logging level",
     show_default=True,
 )
+@click.option(
+    "--rules-config-file",
+    type=click.File("r"),
+    help="Loads rules configuration file for filtering (type: [.py, .pyc])",
```

Comment:
```suggestion
    help="Loads rules configuration file (type: [.py, .pyc])",
```
It also setups rule modes, it's not just filtering